build_tools/prep_kolibri_dist.py

```diff
@@ -53,10 +53,14 @@ def preseed_kolibri(whl):
     subprocess.call(['pip', 'install', '--target', kolibri_env, whl])
     port = 16294
     kolibri_bin = 'tmpenv/bin/kolibri'
-    subprocess.Popen([kolibri_bin, 'start', '--port={}'.format(port)], env=env)
+    kolibri_proc = subprocess.Popen([kolibri_bin, 'start', '--port={}'.format(port)], env=env)
     wait_for_kolibri(port)
 
-    subprocess.check_call([kolibri_bin, 'stop'], env=env)
+    try:
+        subprocess.check_call([kolibri_bin, 'stop'], env=env)
```

Comment:
I was getting shutdown errors that looked like zombie process errors with 0.15, so I updated the code to do a hard shutdown in this case.